sdcm/utils/version_utils.py

```diff
@@ -202,6 +202,7 @@ def is_enterprise(version):
     :param version: version string
     :return: True if this version string passed is a scylla enterprise version
     """
+    assert re.match(r'(\d+.\d+)', version)
```

Comment:
Why do we need this assert? Can we remove it? Unit tests fail because of it.

Reply:
The assert is too strict, '666.development' will fail.
I will remove it.